crates/sui-core/src/unit_tests/data/random/sources/random.move

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
nit, for my sanity when finding files, can we name this something like `uses_random.move` lol
Not a big deal either way